manager/main.go

```diff
@@ -72,7 +72,13 @@ func run(namespace string, metricsAddr string, enableLeaderElection bool,
 		&corev1.PersistentVolumeClaim{}: {Field: workerNamespaceSelector},
 	}
 
-	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
+	client := ctrl.GetConfigOrDie()
+	client.QPS = utils.GetEnvAsFloat32("CLIENT_QPS", 100.0)
```

Comment:
Should we keep the defaults as the original? To avoid suggesting specific values